src/main/scala/io/joern/typestubs/Main.scala

```diff
@@ -1,11 +1,71 @@
 package io.joern.typestubs
 
 import io.joern.typestubs.ruby.BuiltinPackageDownloader
+import scopt.OParser
+
+final case class Config(writeToJson: Boolean = false, languageFrontend: String = "all") {
+
+  def writeToJson(value: Boolean): Config = {
+    copy(writeToJson = value)
+  }
+
+  def withLanguageFrontend(value: String): Config = {
+    copy(languageFrontend = value)
+  }
+}
+
+private object Frontend {
+
+  implicit val defaultConfig: Config = Config()
+
+  val cmdLineParser: OParser[Unit, Config] = {
+    val availableFrontendLanguages = Seq("", "ruby", "chsarp", "all")
+
+    val builder = OParser.builder[Config]
+    import builder.*
+    OParser.sequence(
+      programName("joern-type-stubs"),
+      opt[Unit]("writeToJson")
```

Comment:
We could generalize this to multiple tiers
```
implicit val outputFormatR: scopt.Read[OutputFormat] =
  scopt.Read.reads(OutputFormat.Value.withName _)
...
opt[OutputFormat]('f', "format")
 .action((x, c) => c.withFormat(x))
...
object OutputFormat extends Enumeration {
  val json, mpk, zip = Value
}
```
Default can be `zip`, where all `mpk`s are zipped.